sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -134,7 +134,7 @@ internal ConfigurationClient(Uri endpoint, TokenCredential credential, string sy
         private static HttpPipeline CreatePipeline(ConfigurationClientOptions options, HttpPipelinePolicy authenticationPolicy, HttpPipelinePolicy syncTokenPolicy)
         {
             return HttpPipelineBuilder.Build(options,
-                new HttpPipelinePolicy[] { new CustomHeadersPolicy() },
+                new HttpPipelinePolicy[] { },
```

Comment:
Any chance we can remove this parameter?   If not, maybe we should go with `Array.Empty<HttpPipelinePolicy>()`.